pennylane/templates/layers/particle_conserving_u1.py

```diff
@@ -0,0 +1,253 @@
+# Copyright 2018-2020 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+r"""
+Contains the hardware efficient ``ParticleConservingU1`` template.
+"""
+import numpy as np
+
+import pennylane as qml
+
+# pylint: disable-msg=too-many-branches,too-many-arguments,protected-access
+from pennylane.templates.decorator import template
+from pennylane.ops import CNOT, CRot, PhaseShift, CZ
+from pennylane.templates.utils import (
+    check_shape,
+    get_shape,
+)
+from pennylane.wires import Wires
+
+
+def decompose_ua(phi, wires=None):
+    r"""Implement the circuit decomposing the controlled application of the unitary
+    :math:`U_A(\phi)`
+
+    .. math::
+
+        U_A(\phi) = \left(\begin{array}{cc} 0 & e^{-i\phi} \\ e^{-i\phi} & 0 \\ \end{array}\right)
+
+    in terms of the quantum operations supported by PennyLane.
+
+    :math:`U_A(\phi)` is used in `arXiv:1805.04340 <https://arxiv.org/abs/1805.04340>`_),
```

Comment:
```suggestion
    :math:`U_A(\phi)` is used in `arXiv:1805.04340 <https://arxiv.org/abs/1805.04340>`_,
```
I think that parenthesis is not necessary